analysis_module/serializers.py

```diff
@@ -66,3 +67,26 @@ class PredictionRequestSerializer(serializers.Serializer):
     authoring_organization = serializers.CharField()
     callback_url = serializers.CharField()
     mock = serializers.BooleanField(default=False)
+
+
+class ExtractionDocumentSerializer(serializers.Serializer):
+    url = serializers.CharField()
+    client_id = serializers.CharField()
+
+
+class ExtractionRequestTypeChoices(models.IntegerChoices):
```

Comment:
```suggestion
class ExtractionRequestTriggerTypeChoices(models.IntegerChoices):
```
Can this be a better naming?

Reply:
Looks okay to me.